src/TeamCitySharp/ActionTypes/BuildQueue.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections.Generic;
+using TeamCitySharp.Connection;
+using TeamCitySharp.DomainEntities;
+using TeamCitySharp.Locators;
+
```

Comment:
Normally, I use 2 space not 4 space... I will change this before to publish a new version

Reply:
Great thanks, can you give a hint as to when the nuget package will be updated?

Reply:
Very soon!!

Reply:
done. We need to wait that nuget.org authorize the package

https://www.nuget.org/packages/TeamCitySharp-forked-mavezeau/1.0.58

Reply:
Cool, thanks!